app/src/main/java/com/bielczy/mvvmclean/core/app/roomDataBase/EpisodesDB.kt

```diff
@@ -0,0 +1,68 @@
+package com.bielczy.mvvmclean.core.app.roomDataBase
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import androidx.sqlite.db.SupportSQLiteDatabase
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+
+@Database(entities = [Episodes::class], version = 1)
+abstract class EpisodesDB : RoomDatabase() {
+
+    abstract fun episodesDao(): EpisodesDao
+
+    companion object {
```

Comment:
Ten cały companion object nie będzie nam potrzebny. Widzę, że zadeklarowałeś już bazę danych w `databaseModule` i to tak naprawdę na ten moment wystarczy :)